pkg/liquidity-mining/contracts/VotingEscrowDelegationProxy.sol

```diff
@@ -67,6 +67,14 @@ contract VotingEscrowDelegationProxy is SingletonAuthentication {
         return _adjustedBalanceOf(user);
     }
 
+    /**
+     * @notice Get the current veBAL total supply from the votingEscrow contract.
+     * @return The current veBAL total supply.
+     */
+    function totalSupply() external view returns (uint256) {
+        return _votingEscrow.totalSupply();
+    }
```

Comment:
This seems wrong, shouldn't this call `_totalSupply()`?

Reply:
Whoops, yes that's right.